tests/unit/test_oneview.py

```diff
@@ -2032,6 +2033,17 @@ def test_sort_by_keys(self):
                          dict(networkType="Ethernet", name="name-2")]
         assert result1 == expected_list
 
+    def test_merge_dict_with_single_element_list_inside_dict(self):
+        original_dict = dict(test1=[dict(id=2, allocatedMbps=1000, mac="E2:4B:0D:30:00:0B", requestedMbps=1000)])
+
+        dict_with_changes = dict(test1=[dict(id=2, requestedMbps=2700, allocatedVFs=3500)])
```

Comment:
not seeing all other test cases like new record added/deleted in the list?
Also we should have negative test cases where first list with non empty dict and second list with empty dict